test/http-smoke.js

```diff
@@ -0,0 +1,211 @@
+import _ from 'underscore';
+import chai from 'chai';
+import should from 'should';
+import * as es from 'event-stream';
+import {
+  fixture, bufferJs
+}
+from './fixture';
+import request from 'request';
+import CoreMock from './services/mock-core';
+import MockZKClient from './services/mock-zk';
+import {
+  EventEmitter
+}
+from 'events';
```

Comment:
would prefer if this were all on one line. it's not like you're gonna import a dozen names… makes it harder to do a quick scan of what libraries are being imported, because lib names are no longer the furthest-right thing.

Do ES6 people on the interwebz use this style?